bricks/aem-hero/aem-hero.js

```diff
@@ -2,10 +2,10 @@ import { Brick } from '../../scripts/aem.js';
 
 export default class Hero extends Brick {
   connectedCallback() {
-    const image = this.root.querySelector('picture');
-    const title = this.root.querySelector('h1');
+    const image = this.querySelectorFromRoot('picture');
```

Comment:
I'm not sure this is really useful, it's more characters to type ;-)
Kidding aside, it hides the existing of `this.root` but isn't that an important concept for Bricks? If yes I would prefer omitting this glue method. But it's a question of taste, not an actual objection.